.devcontainer/Dockerfile

```diff
@@ -3,4 +3,10 @@ ARG VARIANT=3-bullseye
 FROM mcr.microsoft.com/vscode/devcontainers/python:${VARIANT}
```

Comment:
@martgra Ser vi bruker variant = 3-bullseye. Er det en grunn til at vi ikke bruker 3.9 eller noe tilsvarende? I tillegg, defineres nå Python versjon både her og i .toml, men kanskje ikke så lett å synce dem

Reply:
Godt poeng. Det kan kanskje være greit å spesifisere både python og debian-versjon. Spørsmålet er hvordan vi håndterer dette i .toml. Ser for meg a) spesifisere versjon 3.X i toml eksakt, eller b) minimum versjonen vi spesifiserer i Dockerfile. type >=3.9

Reply:
I tillegg nå defineres 3-Bullseye i både Dockerfilen og i devcontainer.json (betyr det at devcontainer versjon ikke benyttes?)

Reply:
Jeg er litt usikker - dette er jo slik MS shipper default --> https://github.com/microsoft/vscode-dev-containers/blob/main/containers/python-3/.devcontainer/Dockerfile

Hvis jeg måtte tippe så må VARIANT deklareres med en default-verdi i Dockerfile og kan overstyres fra devcontainer.json.

Reply:
Jeg landet også på å pinne repoet til python 3.9.